### ISSUE TITLE
Paratrooper bug

### ISSUE THREAD

**Issue by panther2** [triplea-game/triplea#1359]
Reported on tripleadev:
http://tripleadev.1671093.n2.nabble.com/World-War-II-v4-td7594489.html

Looks like at any time during the 1.9 process the paratrooper technology has been broken.
I reproduced this issue using wwII_v4 (activating technology in the options).

In 1.9.0.0_3331 infantry cannot be "loaded" onto a bomber. The activated technology has no effect.
It works in 1.8.0.9 - here the respective dialogue is shown and the movement can be done.

As said in the linked thread, other maps are affected, too.

**Comment by simon33-2** [comment]
I see the problem. The Paratrooper functionality appears to only support 1940 type paratroopers which can only take off from airbases. Maps without airbases need another way of doing it. aa50 is a rare example of a board game with these rules.

**Comment by ron-murhammer** [comment]
@simon33-2 True but I believe this worked in 1.8.0.9 so something had to have changed.

**Comment by DanVanAtta** [comment]
Should now be fixed, quite a number of maps had issues.

https://github.com/triplea-game/triplea/pull/1370

**Comment by simon33-2** [comment]
This works for me on v4 although I've downloaded the map via git.

I'll open a PR for moving on the version number (Europe 1940, Pacific 1940, v3, v4, v6). I'll also update the SBR for v6 if anyone plays that.

In fact, I might as well wait until all the above PRs are merged. I'm happy to do a bulk update if you're happy to give me write access to all the maps. Otherwise it would be a pain for both of us.

**Comment by DanVanAtta** [comment]
Given the problems we've had, I'm even more cautious about bulk updates than before. I did the recent round of paratroop fixes via PRs, was about 20 maps -  it was not that bad.

SBR sounds like it should get patched everywhere, I'm curious if you could give details why the patches proposed work?

**Comment by simon33-2** [comment]
I've determined by testing that in 1.809 where the damage property is not set, it refers to the SBR property. 1.9 doesn't do this. While it is true that we could update the engine to do the same as 1.809, that seems to leave us with a pointless property in the system. Better to update remove the SBR property from the system as I've outlined. Does that make sense now? Seems to be what Cernelius was saying all along.

I think there's 30 something maps involved in the SBR. Looks like there is a way of auto logging PRs so I can do that if you want.
